server/services/prisonerProfileService.test.ts

```diff
@@ -445,60 +409,62 @@ describe('Prisoner profile service', () => {
+          latestPrivIepAdjustDate: '2021-12-01',
+        },
+        visits: [],
       }
 
-      const inmateDetail = TestData.inmateDetail({
-        activeAlertCount: 4,
-        inactiveAlertCount: 1,
-        alerts: [inactiveAlert, nonRelevantAlert, ...alertsToFlag],
-        legalStatus: 'REMAND',
-      })
+      fullPrisoner.alerts = [inactiveAlert, nonRelevantAlert, ...alertsToFlag]
 
-      const prisoner = TestData.prisoner()
+      orchestrationApiClient.getPrisonerProfile.mockResolvedValue(fullPrisoner)
 
-      prisonApiClient.getBookings.mockResolvedValue(bookings)
-      prisonApiClient.getOffender.mockResolvedValue(inmateDetail)
-      prisonerSearchClient.getPrisonerById.mockResolvedValue(prisoner)
-      visitSchedulerApiClient.getUpcomingVisits.mockResolvedValue({ content: [] })
-      visitSchedulerApiClient.getPastVisits.mockResolvedValue({ content: [] })
+      const results = await prisonerProfileService.getProfile(prisonId, prisonerId, 'user')
 
-      const results = await prisonerProfileService.getProfile(offenderNo, prisonId, 'user')
+      expect(OrchestrationApiClientFactory).toHaveBeenCalledWith(token)
+      expect(hmppsAuthClient.getSystemClientToken).toHaveBeenCalledWith('user')
+      expect(orchestrationApiClient.getPrisonerProfile).toHaveBeenCalledTimes(1)
+      expect(supportedPrisonsService.getSupportedPrisons).toHaveBeenCalledTimes(1)
 
-      expect(prisonApiClient.getBookings).toHaveBeenCalledTimes(1)
-      expect(prisonApiClient.getOffender).toHaveBeenCalledTimes(1)
-      expect(prisonApiClient.getVisitBalances).not.toHaveBeenCalled()
-      expect(prisonerSearchClient.getPrisonerById).toHaveBeenCalledTimes(1)
-      expect(prisonerContactRegistryApiClient.getPrisonerSocialContacts).toHaveBeenCalledTimes(1)
-      expect(results).toEqual(<PrisonerProfile>{
-        displayName: 'Smith, John',
-        displayDob: '2 April 1975',
+      expect(results).toEqual(<PrisonerProfilePage>{
         activeAlerts: alertsForDisplay,
+        activeAlertCount: 4,
         flaggedAlerts: alertsToFlag,
-        inmateDetail,
-        convictedStatus: 'Remand',
-        incentiveLevel: 'Standard',
-        visitBalances: null,
-        upcomingVisits: [],
-        pastVisits: [],
+        visits: [],
+        prisonerDetails: {
+          offenderNo: 'A1234BC',
+          name: 'Smith, John',
+          dob: '2 April 1975',
+          convictedStatus: 'Convicted',
+          category: 'Cat C',
+          location: '1-1-C-028',
+          prisonName: 'Hewell (HMP)',
+          incentiveLevel: 'Standard',
+          visitBalances: {
+            remainingVo: 1,
+            remainingPvo: 2,
+            latestIepAdjustDate: '21 April 2021',
+            latestPrivIepAdjustDate: '1 December 2021',
+          },
+        },
       })
     })
 
-    it('Throws 404 if no bookings found for criteria', async () => {
+    it.skip('Throws 404 if no bookings found for criteria', async () => {
```

Comment:
We need to test how it handles a 404 from the orchestration service (assuming that that **is** what it returns if a prisoner can't be found). Can sort as a separate PR though.